lib/util/nested_struct.rb

```diff
@@ -16,7 +16,7 @@ def initialize(json=Hash.new)
                               value
                             end
 
-      new_ostruct_member(key)
+      new_ostruct_member!(key)
```

Comment:
Just curious why this needs the bang method. (Full disclosure, I'm not a Ruby developer).

Reply:
It's default library from ruby
It was changed in Ruby 2.4.0 https://docs.ruby-lang.org/en/2.4.0/ChangeLog-2_4_0.html#label-2016-03-03